src/window/window_wrapper/mod.rs

```diff
@@ -322,7 +322,14 @@ impl GlutinWindowWrapper {
                     },
                 ..
             } => self.handle_mouse_wheel(x as i32, y as i32),
-
+            Event::WindowEvent {
+                event:
+                    WindowEvent::MouseWheel {
+                        delta: MouseScrollDelta::PixelDelta(lpos),
+                        ..
+                    },
+                ..
+            } => self.handle_mouse_wheel(0, (lpos.y as f32)/10),
```

Comment:
Cast lpos.y to f32 instead of i32 (draft), resolves ignoring small inputs (hopefully)